tests/stdlib/tos.nim

```diff
@@ -651,3 +651,7 @@ block: # normalizeExe
     doAssert "foo/../bar".dup(normalizeExe) == "foo/../bar"
   when defined(windows):
     doAssert "foo".dup(normalizeExe) == "foo"
+
+block: # isAdmin
+  let isAzure = defined(windows) and existsEnv("TF_BUILD") # xxx factor with testament.specs.isAzure
+  if isAzure: doAssert isAdmin()
```

Comment:
```suggestion
  let isAzure = existsEnv("TF_BUILD") # xxx factor with testament.specs.isAzure
  if isAzure:
    when defined(windows): doAssert isAdmin()
```
because you can have azure for non-windows IIUC

Reply:
It doesn't work: it looks like in Azure on POSIX tests are launched as a non-root user.

Reply:
Fixed it another way.